hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,3 @@
+RELEASE_TYPE: minor
+
+**This is a placeholder to check if the location for this file is correct**
```

Comment:
It is correct, but you'll need some actual release notes here 😉 

See [changes.rst](https://github.com/HypothesisWorks/hypothesis/blob/master/hypothesis-python/docs/changes.rst) or PR #3285 for examples; and please add yourself to the list in `AUTHORS.rst` too!